web-api/src/business/useCases/processStreamRecords/processCaseEntries.ts

```diff
@@ -5,6 +5,7 @@ import { upsertCaseStatusUpdates } from '@web-api/persistence/postgres/cases/ups
 import { Statistic } from '@shared/business/entities/Statistic';
 import { unmarshall } from '@aws-sdk/util-dynamodb';
 import { settlePromises } from '@web-api/utilities/settlePromises';
+import { getLogger } from '@web-api/utilities/logger/getLogger';
```

Comment:
The diffs look more confusing than they are. I have just wrapped everything in a try/catch, which throws off indentation.